message_app_server/message_app/user_sign_in.py

```diff
@@ -35,14 +35,18 @@ def api_user_signin():
 
         if check_pw(user_password_input, user_password_hash):
             session["user"] = (user_data.id, username)
-
-            return redirect("http://localhost:3000/messages")
+            payload = {
+                "status": "Successful",
+                "sessionCookie": "",
```

Comment:
So we would need to think about how we send the session cookie in the payload, right?

Reply:
Yes